arangod/Aql/Projections.h

```diff
@@ -93,6 +95,9 @@ class Projections {
   /// @brief reset all projections
   void clear() noexcept;
 
+  /// @brief erase projection at index
+  void erase(size_t index);
```

Comment:
I think better to make method that accept boolean filter.
So this method can use std::remove_if to avoid quadratic complexity for cycles with erase

Reply:
Changed